Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla41619.cs

```diff
@@ -32,15 +39,15 @@ protected override void Init()
 				{
 					label,
 					slider,
-					new Label { Text = "The initial slider value above should be 5." }
+					new Label { Text = "The initial slider value above should be 6." }
```

Comment:
Instead of hardcoding 6, you could use `_success` so that the Label text is dynamic.